### ISSUE TITLE
New lint suggestion: Remove commented code

### ISSUE THREAD

**Issue by yonip23** [rust-lang/rust-clippy#6707]
### What it does

Checks for commented code - possibly should be deleted

### 

- Kind: correctness / style

Advantages:
- Get rid of redundant and confusing code that doesn't even compile
- Clean up the codebase.

### Drawbacks

None.

### Example

```rust
// fn foo() {
// ...
//}
```

Should be deleted.

**Comment by giraffate** [comment]
Thanks for new lint suggestion! It seems to be a duplicate of https://github.com/rust-lang/rust-clippy/issues/1348, so I'm closing this. If you have anything on this, feel free to reopen this.
